Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionVC.swift

```diff
@@ -333,11 +333,17 @@ class EnhancedTrackingProtectionMenuVC: UIViewController {
     }
 
     private func updateViewDetails() {
+        heroImage.image = UIImage(named: "defaultFavicon")!
```

Comment:
nit, doesn't have to be forced
```suggestion
        heroImage.image = UIImage(named: "defaultFavicon")
```